gst/nnstreamer/include/nnstreamer_plugin_api_filter.h

```diff
@@ -21,7 +21,10 @@
 #define ACCL_DEFAULT_STR  "default"
 #define ACCL_AUTO_STR "auto"
 #define ACCL_CPU_STR  "cpu"
+#define ACCL_CPU_SIMD_STR  "cpu.simd"
+#if defined(__aarch64__) || defined(__arm__)
```

Comment:
This imposes a restriction on subplugins to not add NEON in supported accelerators list in non-arm architectures.

Reply:
This is really a wild assumption, but we may still need to designate NEON in a filter with x64 host; an NPU with ARM core installed in x64 machine. Or in a integrated multi-node pipeline, we may need to say "NEON" from x64 machine where a part of a pipeline might be executed in another node.

We can keep CPU_NEON_STR definition for all arch. We just need to block auto-translation of SIMD -> NEON or NEON -> SIMD in non ARM.

Reply:
The assumption is removed.